lib/hooks/useCustomers.ts

```diff
@@ -12,11 +12,25 @@ import { NEXT_PUBLIC_API_URL } from "../conf"
 import { User } from "firebase/auth"
 
 export const customerEndpointURL = `${NEXT_PUBLIC_API_URL}/customer`
```

Comment:
For reasons I will explain later this file will probably need a `matchMutate`. Take a look at the `lib/hooks/useProjects.ts` file and look for match mutate.

You should define 
```
const projectIdEndpointCacheKeyRegex = /^\/project\/([0-9]+)$/
```
here and have the corresponding `matchMutate` for put and post hooks in this file below.